packages/iframe-coordinator/package.json

```diff
@@ -19,21 +19,20 @@
     "@rollup/plugin-replace": "^5.0.5",
     "@rollup/plugin-typescript": "^11.1.5",
     "@types/jasmine": "^5.1.4",
-    "cross-var": "^1.1.0",
     "jasmine-core": "^5.1.1",
     "karma": "^6.4.2",
     "karma-chrome-launcher": "^3.2.0",
     "karma-jasmine": "^5.1.0",
     "karma-typescript": "^5.5.4",
-    "puppeteer": "^21.6.1",
-    "rollup": "^4.9.1",
-    "typedoc": "^0.25.4",
+    "puppeteer": "^21.7.0",
+    "rollup": "^4.9.4",
+    "typedoc": "^0.25.7",
     "typescript": "^4.9.5"
   },
   "scripts": {
     "build": "npm run build-lib",
     "build-lib": "rollup -c",
-    "current-version": "cross-var \"echo $npm_package_version\"",
+    "current-version": "echo $npm_package_version",
```

Comment:
I removed `cross-var` as it is an unmaintained package that has a dependency with a vulnerability. It is also overkill for an echo command that is basically already cross platform.